### DISCUSSION TITLE
ERGM in constrained setting

### DISCUSSION THREAD

**Discussion by woisokoh** [statnet/ergm#553] [Q&A]
Hi,

I would like to use ERGM for my network. This network is location-based at the US tract level, and I constrained connections with other tracts whose borders are shared with the focal tract (node). I have a list of connection candidates (neighbors) for each node. I was wondering how can I reflect this situation into the ERGM development. The below is how the network looks (red is one-directional, yellow is bi-directional; width stands for different strengths but please forget about it since I am gonna work with unweighted one).
![dada](https://github.com/statnet/ergm/assets/91989726/3c642721-1673-44ce-9562-6ca0f1073193)

**Comment by krivit** [comment]
This looks like a dyadic constraint. Try `?ergmConstraint` to see if any of those will do the trick.
